config/authorities.yml

```diff
@@ -1 +1,5 @@
-:local_path: "config/authorities"
\ No newline at end of file
+:local_path: "config/authorities"
+academic_units:
+  uri: 'http://opaquenamespace.org/ns/osuAcademicUnits.jsonld'
+  cache_expires_in: 10.minutes
```

Comment:
I kinda like the default of `1.day` or `12.hours` or so.. that way our app isn't query ONS a whole lot because in general that vocabulary isn't going to get updated.